tests/trx_generator/main.cpp

```diff
@@ -45,33 +45,26 @@ struct signed_transaction_w_signer {
    fc::crypto::private_key _signer;
 };
 
-vector<action_pair_w_keys> create_initial_transfer_actions(const std::string& salt, const uint64_t& period, const name& newaccountT, const vector<name>& accounts, const vector<fc::crypto::private_key>& priv_keys, const fc::microseconds& abi_serializer_max_time) {
-   vector<action_pair_w_keys> actions_pairs_vector;
+chain::bytes make_transfer_data(const chain::name& from, const chain::name& to, const chain::asset& quantity, const std::string&& memo) {
+   return fc::raw::pack<chain::name>(from, to, quantity, memo);
+}
 
-   abi_serializer eosio_token_serializer{fc::json::from_string(contracts::eosio_token_abi().data()).as<abi_def>(), abi_serializer::create_yield_function(abi_serializer_max_time)};
+auto make_transfer_action(chain::name account, chain::name from, chain::name to, chain::asset quantity, std::string memo) {
+   return chain::action(std::vector<chain::permission_level>{{from, chain::config::active_name}},
+                        account, "transfer"_n, make_transfer_data(from, to, quantity, std::move(memo)));
+}
+
+vector<action_pair_w_keys> create_initial_transfer_actions(const std::string& salt, const uint64_t& period, const name& newaccountT, const vector<name>& accounts, const vector<fc::crypto::private_key>& priv_keys) {
+   vector<action_pair_w_keys> actions_pairs_vector;
 
    for(size_t i = 0; i < accounts.size(); ++i) {
       for(size_t j = i + 1; j < accounts.size(); ++j) {
          //create the actions here
          ilog("create_initial_transfer_actions: creating transfer from ${acctA} to ${acctB}", ("acctA", accounts.at(i))("acctB", accounts.at(j)));
-         action act_a_to_b;
-         act_a_to_b.account = newaccountT;
-         act_a_to_b.name = "transfer"_n;
-         act_a_to_b.authorization = vector<permission_level>{{accounts.at(i), config::active_name}};
-         act_a_to_b.data = eosio_token_serializer.variant_to_binary("transfer",
-                                                                    fc::json::from_string(fc::format_string("{\"from\":\"${from}\",\"to\":\"${to}\",\"quantity\":\"1.0000 CUR\",\"memo\":\"${l}\"}",
-                                                                    fc::mutable_variant_object()("from", accounts.at(i).to_string())("to", accounts.at(j).to_string())("l", salt))),
-                                                                    abi_serializer::create_yield_function(abi_serializer_max_time));
+         action act_a_to_b = make_transfer_action(newaccountT, accounts.at(i), accounts.at(j), asset::from_string("1.0000 CUR"), salt);
```

Comment:
We exclusively use camel_case except for template types.

Reply:
I meant snake_case.

Reply:
Addressed: fb0dd3ada02ac8b36f7225bc54008176fbd64ce2